src/Tribe/RSVP.php

```diff
@@ -655,7 +655,16 @@ public function generate_tickets( ) {
 		}
 	}
 
-	public function send_tickets_email( $order_id ) {
+	/**
+	 * Dispatches a confirmation email that acknowledges the user has RSVP'd
+	 * including the tickets.
+	 *
+	 * @since TBD added $event_id parameter
+	 *
+	 * @param int $order_id
+	 * @param int $event_id
+	 */
+	public function send_tickets_email( $order_id, $event_id ) {
```

Comment:
I should have flagged this before, but we should default to `$event_id` being null (ie, `send_tickets_email( $order_id, $event_id = null )`).

Reason is this is a public method that existing 3rd party code may call: if those calls don't provide both parameters, we'll have breakages: giving `$event_id` a default though will resolve that.